### PR TITLE
Add `query-scheduler.libsonnet`

### PR DESCRIPTION
**Pull request by pstibrany**
**What this PR does**: This PR adds `query-scheduler.libsonnet` for configuring query-scheduler.

**Checklist**
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by pracucci — COMMENTED** [review]
Instead of having to include the source file to enable it, can we just have a `cortex_query_scheduler_enabled` in the config? I think it would be more practical to enable/disable.

**Comment by pstibrany** [comment]
> Instead of having to include the source file to enable it, can we just have a `cortex_query_scheduler_enabled` in the config? I think it would be more practical to enable/disable.

You're right. Please take a look again.

File: cortex/images.libsonnet [hunk 1]
```diff
@@ -17,6 +17,7 @@
     flusher: self.cortex,
     ruler: self.cortex,
     store_gateway: self.cortex,
+    query_scheduler: $._images.cortex,
```
Comment (pracucci) [inline cortex/images.libsonnet:20]:
```suggestion
    query_scheduler: self.cortex,
```

**Review by pracucci — APPROVED** [review]
Thanks for working on it! LGTM (modulo a nit) 🙏

File: cortex/images.libsonnet (hunk 1 above)
Comment (pstibrany) [reply cortex/images.libsonnet]: 🤦 Thanks!

Comment (pstibrany) [reply cortex/images.libsonnet]: (Although it worked the other way too ;-))

**Pull request merged**